eclass/qt5-build.eclass

```diff
@@ -136,6 +136,17 @@ qt5-build_src_unpack() {
 qt5-build_src_prepare() {
 	qt5_prepare_env
 
+	if [[ -n ${KDE_ORG_COMMIT} ]]; then
+		# Upstream bumped version in 5.15 branch after 5.15.2 release but their
+		# 5.15.3 release is closed and this will never be more than a Qt 5.15.2
+		# with patches on top.
+		einfo "Preparing KDE Qt5PatchCollection snapshot at ${KDE_ORG_COMMIT}"
+		sed -e "/^MODULE_VERSION/s/5\.15\.3/5\.15\.2/" -i .qmake.conf || die
+		if ! [[ -d .git ]]; then
```

Comment:
no need for this conditional if you're using `mkdir -p` later, or am I missing something?